pages/linux/iwconfig.md

```diff
@@ -0,0 +1,20 @@
+# ifconfig
```

Comment:
```suggestion
# iwconfig
```

Reply:
Ups! A little mistake haha. I am going to update the PR according with your suggestion. Thank you, @pxgamer!